sbt-version-policy/src/main/scala/sbtversionpolicy/SbtVersionPolicyMima.scala

```diff
@@ -15,7 +15,6 @@ object SbtVersionPolicyMima extends AutoPlugin {
   object autoImport {
     val versionPolicyPreviousVersions = settingKey[Seq[String]]("Previous versions to check compatibility against.")
     val versionPolicyFirstVersion = settingKey[Option[String]]("First version this module was or will be published for.")
-    val versionScheme = settingKey[Option[String]]("Version scheme used for this build: early-semver, pvp, semver-spec")     
```

Comment:
One of the motivation of using `Option[String]` (as opposed to case object etc) was so we could define it like this without depending on sbt 1.4.0.

Reply:
Do you think it would be better to downgrade our dependency on sbt? The problem is that if a user does use sbt 1.4.x, then he gets an error message saying that the setting is defined twice.

Reply:
> The problem is that if a user does use sbt 1.4.x, then he gets an error message saying that the setting is defined twice.

Ah. Maybe we just need to get it out of autoImport to avoid that variable getting into the scope twice.

Reply:
I think it’s fine to ask users to upgrade to 1.4.x